lib/gateway/Shard.js

```diff
@@ -2338,11 +2338,20 @@ class Shard extends EventEmitter {
                 /**
                 * Fired when an interaction is created
                 * @event Client#interactionCreate
-                * @prop {PingInteraction | CommandInteraction | ComponentInteraction | AutocompleteInteraction | UnknownInteraction} Interaction The Interaction that was created
+                * @prop {PingInteraction | CommandInteraction | ComponentInteraction | AutocompleteInteraction | UnknownInteraction} interaction The Interaction that was created
                 */
                 this.emit("interactionCreate", Interaction.from(packet.d, this.client));
                 break;
             }
+            case "APPLICATION_COMMAND_PERMISSIONS_UPDATE": {
+                /**
+                 * Fired when an application command's permissions are updated
+                 * @event Client#applicationCommandPermissionsUpdate
+                 * @prop {GuildApplicationCommandPermissions} guildApplicationCommandPermissions The updated command permissions
```

Comment:
```suggestion
                 * @prop {Object} guildApplicationCommandPermissions The updated command permissions
```
Can't use typescript interfaces in JSDoc